adapters/infytv/infytv.go

```diff
@@ -0,0 +1,78 @@
+package infytv
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/mxmCherry/openrtb/v15/openrtb2"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+type InfytvAdapter struct {
```

Comment:
You can call this simply "Adapter", the "Infytv" identification is already supplied by the package name. As you have it, referencing your adapter from outside the package would be `infytv.InfytvAdapter` which looks a little redundant.

Reply:
@hhhjort Thanks for the suggestion. Will update it.